src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -1023,3 +1027,20 @@ def alertDatatierNotFound(self, datatierName, containerName, isRelVal):
         self.logger.critical(alertDescription)
         if self.msConfig["sendNotification"] and not isRelVal:
             self.sendAlert(alertName, alertSeverity, alertSummary, alertDescription, self.alertServiceName)
+
+    def alertGenericError(self, caller, workflowname, msg, ex_str, document_str):
+        """
+        Send an alert in the case of a generic error with ms-output 
+        :param msg: string
+        :return: none
+        """
+        alertName = "ms-output: Generic MSOutput error inside {} while processing workflow '{}'".format(caller, workflowname)
+        alertSeverity = "high"
+        alertSummary = "[MSOutput] Generic MSOutput error inside {} while processing workflow '{}'".format(caller, workflowname)
+        alertDescription = "wf: {}\n\nmsg: {}\n\nex: {}\n\n{}".format(workflowname, msg, ex_str, document_str)
+        self.logger.error("%s\n%s\n%s", alertName, alertSummary, alertDescription)
+        if self.msConfig["sendNotification"]:
+            self.sendAlert(alertName, alertSeverity, alertSummary, alertDescription, self.alertServiceName)
```

Comment:
Should we explicitly set an expiration time for this alert? Without it, what is the default value?

Reply:
Thank you, Dario. Where did you come with `repeat_interval` configuration from?

I am undecided whether we should make this time for re-raising an alert larger or not, as a fear of spam. 
Would 12h be a better option?

Reply:
it's always the same file, [here](https://gitlab.cern.ch/cmsmonitoring/cmsmon-configs/-/blob/3997741ec507dcd99901591e80618d19b7672edb/alertmanager/alertmanager.yaml#L13)

we can change to 12h if:

- we create a new receiver for alertmanager (for example `dmwm-admins-12h`) , and we override the default values (for example setting 12h for repeat interval)
- in msoutput.py, we override the value for `tag`, let's say `wmcore12h`
- we create a new redirect for alertmanager that matches the tag `wmcore12h` and send the alert to `dmwm-admins-12h`

at this point it would be beneficial to have a broader discussion on our alerts, because we could take this opportunity to improve the situation across the board.

Reply:
Thank you for these details.
Given that we would have to either change the default repeat_interval value or fork it for `dmwm`, I would suggest to leave it for one of the monitoring-related issues that we are discussing and considering for Q1. I fully agree that a discussion on that is important, so I suggest to keep it out of these developments.